roles/sneakers_worker/meta/main.yml

```diff
@@ -7,13 +7,13 @@ galaxy_info:
 
   license: MIT
 
-  min_ansible_version: 2.4.0.0
+  min_ansible_version: "2.4.0.0"
 
   platforms:
     - name: Ubuntu
       versions:
-        - 16.04
-        - 18.04
+        - xenial  
```

Comment:
Xenial is EOL, so we can remove this line. It looks like this is also the source of the "trailing whitespace" error in CI.